frontend/ui/src/app/components/side-bar/side-bar.component.html

```diff
@@ -5,6 +5,23 @@
   #asideElement>
   <div class="h-full flex flex-col px-3 py-2 gap-2 overflow-y-auto bg-white dark:bg-gray-800">
     <ul class="space-y-2 font-medium flex-1" *appRequiredRole="'vendor'">
+      <li>
+        <a
+          (click)="sidebar.hide()"
+          routerLinkActive="bg-gray-100 dark:bg-gray-700"
+          routerLink="/tutorials"
+          class="flex items-center p-2 text-gray-900 rounded-lg dark:text-white hover:bg-gray-100 dark:hover:bg-gray-700 group">
+          <fa-icon
+            [icon]="faLightbulb"
+            size="lg"
+            class="pl-0.5 w-6 h-6 text-gray-400 transition duration-75 dark:text-gray-400 group-hover:text-gray-900 dark:group-hover:text-white"></fa-icon>
+          <span class="flex-grow ms-3">Tutorials</span>
+          @if (tutorialsService.notAllStarted$ | async) {
+            <span
+              class="w-2.5 h-2.5 mr-1.5 text-xs font-semibold rounded-full text-primary-800 bg-primary-100 dark:bg-primary-200 dark:text-primary-800"></span>
```

Comment:
I think this indicator could be made more prominent by changing the color. Maybe use a more saturated shade of blue? Or red? In light mode it's currently almost invisible.